editor/src/components/navigator/navigator-item/navigator-item-wrapper.tsx

```diff
@@ -64,7 +65,10 @@ const labelSelector = createSelector(
   (store: MetadataSubstate) => store.editor.allElementProps,
   (elementMetadata, allElementProps) => {
     if (elementMetadata == null) {
-      return 'Element 👻'
+      if (!isFeatureEnabled('Conditional support')) {
+        return 'Element 👻'
+      }
+      return 'Conditional'
```

Comment:
Do we know it's a conditional at this point? Don't we need to actually check that?

Reply:
This was put in place before having conditionals metadata - but now that we have that, this is still the case (for now?) for nested conditionals AFAICT

Reply:
Fixed in 17e7060c229026586bfbcea6304e4408b77a5f97